### PR TITLE
Greately reduces standard holoparasite battlecry spam

### PR DESCRIPTION
**Pull request by Shadowlight213**
number of battlecries per punch reduced to 4
Battlecry will only trigger every 5th punch.
:cl: Shadowlight213
tweak: Standard holoparasite battlecry has been greatly reduced to decrease spam.
/:cl:

### CODE REVIEW

**Comment by Zilenan91** [comment]
This was a long time coming :+1:

**Comment by bgobandit** [comment]
fun removal

File: code/modules/mob/living/simple_animal/guardian/guardian.dm [hunk 1]
```diff
@@ -244,12 +246,14 @@
 /mob/living/simple_animal/hostile/guardian/punch/AttackingTarget()
 	..()
 	if(istype(target, /mob/living))
-		src.say("[src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry]\
-		[src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry]")
+		if(src.cry_wait == 0)
```
Comment (Jordie0608) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: Use world.time for cooldowns

Comment (Shadowlight213) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: The idea was for having it be every 5th punch, but yeah I guess a time based cooldown would be better.

Comment (Jordie0608) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: Weirdness would happen like punching one guy twice, then minutes later punching another and only having the battlecry come on the third punch.

**Comment by KorPhaeron** [comment]
Could at least add !!!! to the end so they YELL IN BOLD LETTERS?

Also every X punches or very short cry would be fine, but is a long cry really bad if its not triggering as often?

**Comment by Shadowlight213** [comment]
@KorPhaeron I'm changing it to be on a time based cooldown.
Also, I might increase the cry size slightly, maybe by one or two, but that's it. It's just pure spam honestly. I'm still tempted to just remove it outright.

File: code/modules/mob/living/simple_animal/guardian/guardian.dm [hunk 2]
```diff
@@ -223,6 +223,8 @@
 /mob/living/simple_animal/hostile/guardian/punch
 	melee_damage_lower = 20
 	melee_damage_upper = 20
+	var cry_cooldown = 5
+	var cry_wait = 0
```
Comment (phil235) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: please use `var/cry_cooldown = 5` and `var/cry_wait = 0`

**Comment by KorPhaeron** [comment]
Since your PR broke and I'm editing this file anyway I'll add your fix to my PR

**Pull request closed without merging**